src/node/desktop/src/main/detect-r.ts

```diff
@@ -278,29 +278,30 @@ function scanForR(): Expected<string> {
 }
 
 function scanForRPosix(): Expected<string> {
-  // first, look for R on the PATH
+  if (process.platform == 'darwin') {
+    // For Mac, we want to look in a list of hard-coded locations
+    const defaultLocations = ['/opt/local/bin/R', '/usr/local/bin/R', '/usr/bin/R'];
```

Comment:
https://github.com/rstudio/rstudio/blob/8df7e8c7c6b7727779786613a9089ba14e0c4a68/src/cpp/core/r_util/REnvironmentPosix.cpp#L109-L115

Qt seems to use this order:
- /usr/bin/R
- /usr/local/bin/R
- /opt/local/bin/R
- /opt/homebrew/bin/R
- /Library/Frameworks/R.framework/Resources/bin/R

Reply:
Yeah, but I think putting homebrew R before Cran R is a big mistake for the majority of our users and is just not a good idea, especially with the M1 macs